src/index.js

```diff
@@ -176,8 +179,11 @@ module.exports = async (options) => {
 
 
                 if (stream) {
+                    // Stream need to piped to PassThrough to stay readable
+                    const passThrough = new PassThrough();
+
                     // Add http.IncomingMessage properties to decompress stream.
-                    return resolve(addResponsePropertiesToStream(decompressedResponse, res));
+                    return resolve(addResponsePropertiesToStream(decompressedResponse.pipe(passThrough), res));
```

Comment:
Could you split this into some vars? 🤣

Reply:
Tbh, I don't get why this works. The response stream is already piped to the `decompress` stream so why does an additional pipe help? Also, we were resuming the `requestStream`, but piping the `response` stream to the pass through ❓

Reply:
It works 🌞